code/game/objects/items/cards_ids.dm

```diff
@@ -661,6 +661,15 @@ update_label("John Doe", "Clowny")
 	access = get_all_accesses()
 	. = ..()
 
+/obj/item/card/id/ert/lawyer
+	registered_name = "CentCom Attorney"
+	assignment = "CentCom Attorney"
+	icon_state = "centcom"
+
+/obj/item/card/id/ert/lawyer/Initialize(mapload)
+	access = list(ACCESS_CENT_GENERAL, ACCESS_COURT, ACCESS_BRIG, ACCESS_FORENSICS_LOCKERS)
```

Comment:
This gets overriden to have all access in ..() anyway

Reply:
Switched the order